### PR TITLE
add ROS 2 Eloquent

### PR DESCRIPTION
**Pull request by dirk-thomas**
Add Eloquent build files with the goal to get CI jobs on build.ros2.org going.

Needs ros/rosdistro#21491.

### CODE REVIEW

**Review by mjcarroll — APPROVED** [review]

**Comment by cottsay** [comment]
@nuclearsandwich, is there cause to hold off on merging this until `ros-eloquent-ros-workspace` has been created?

File: eloquent/release-bionic-arm64-build.yaml [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+jenkins_binary_job_timeout: 720
+jenkins_source_job_priority: 75
+jenkins_source_job_timeout: 30
+notifications:
+  emails:
+  - michael+build.ros2.org@openrobotics.org
+  - ros2-buildfarm-eloquent@googlegroups.com
+  maintainers: true
+sync:
+  package_count: 1
```
Comment (nuclearsandwich) [inline eloquent/release-bionic-arm64-build.yaml]: There's no reason to sync until ros_workspace is built which depends on ament_package and ament_cmake_core.

```suggestion
  package_count: 3
```

File: eloquent/release-build.yaml [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+jenkins_binary_job_timeout: 120
+jenkins_source_job_priority: 75
+jenkins_source_job_timeout: 30
+notifications:
+  emails:
+  - michael+build.ros2.org@openrobotics.org
+  - ros2-buildfarm-eloquent@googlegroups.com
+  maintainers: true
+sync:
+  package_count: 1
```
Comment (nuclearsandwich) [inline eloquent/release-build.yaml]: There's no reason to sync until ros_workspace is built which depends on ament_package and ament_cmake_core.

```suggestion
  package_count: 3
```

**Review by nuclearsandwich — APPROVED** [review]

**Comment by nuclearsandwich** [comment]
> @nuclearsandwich, is there cause to hold off on merging this until `ros-eloquent-ros-workspace` has been created?

We need the release-build configs to create packages. I think it's fine to merge the CI config and then not deploy it until after the release builds are done. I am also fine splitting the PR.

**Review by mjcarroll — APPROVED** [review]

**Pull request merged**